pallets/collator-assignment/src/lib.rs

```diff
@@ -140,6 +140,9 @@ pub mod pallet {
     #[pallet::storage]
     pub(crate) type Randomness<T: Config> = StorageValue<_, [u8; 32], ValueQuery>;
 
+    #[pallet::storage]
+    pub type CollatorFullnessRatio<T: Config> = StorageValue<_, Perbill, OptionQuery>;
```

Comment:
Just nitpicking, but the name is a bit confusing for me. I think it will be more confusing once a collator could be assigned multiple containers. Maybe `AssignmentMaxCollatorsRatio`? (not much better, I know)

Reply:
In that case it makes sense, but as Gorka said, a comment explaining would really help

Reply:
fixed